hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/util/DnsResolver.java

```diff
@@ -4,29 +4,30 @@
 import org.apache.commons.logging.LogFactory;
 import org.apache.mesos.Protos;
 import org.apache.mesos.SchedulerDriver;
-import org.apache.mesos.hdfs.Scheduler;
-import org.apache.mesos.hdfs.config.SchedulerConf;
+import org.apache.mesos.hdfs.scheduler.Scheduler;
+import org.apache.mesos.hdfs.config.HdfsFrameworkConfig;
 
 import java.io.IOException;
 import java.net.InetAddress;
 import java.util.HashSet;
 import java.util.Set;
 import java.util.Timer;
-import java.util.TimerTask;
 
 public class DnsResolver {
-  public static final Log log = LogFactory.getLog(Scheduler.class);
+  private final Log log = LogFactory.getLog(Scheduler.class);
+
+  static final int NN_TIMER_PERIOD = 10000;
 
   private final Scheduler scheduler;
-  private final SchedulerConf conf;
```

Comment:
We need consistent naming. This should be hdfsFrameworkConfig as it is elsewhere